test/distributed/cases/mo_cloud/mo_cloud.sql

```diff
@@ -158,7 +158,7 @@ SELECT count(*) FROM `mo_mo`.`wb_version` WHERE wb_id = 'd457fd59-a865-4a5b-80b0
 SELECT count(*) FROM `mo_mo`.`wb` WHERE account_id = '91731e77_49ea_4a8a_b1c7_d5512c7ae96e' AND sql_user = 'admin' AND id = '30415473-5276-42c5-8302-4e9a7e605900';
 SELECT count(*) FROM information_schema.tables WHERE table_schema = '' AND table_name = 'moins' AND table_type = 'BASE TABLE';
 SELECT datname AS name, IF (table_cnt IS NULL, 0, table_cnt) AS tables, role_name AS owner FROM (SELECT dat_id, datname, mo_database.created_time, IF(role_name IS NULL, '-', role_name) AS role_name FROM mo_catalog.mo_database LEFT JOIN mo_catalog.mo_role ON mo_database.owner = role_id) AS x LEFT JOIN(SELECT count(*) AS table_cnt, reldatabase_id FROM mo_catalog.mo_tables WHERE relkind IN ('r','v','e','cluster') GROUP BY reldatabase_id) AS y ON x.dat_id = y.reldatabase_id;
-SELECT mo_catalog.mo_database.datname,mo_catalog.mo_tables.relname,mo_catalog.mo_tables.relkind, if (role_name IS NULL,'-', role_name) AS `owner` FROM mo_catalog.mo_database LEFT JOIN mo_catalog.mo_tables ON mo_catalog.mo_database.datname = mo_catalog.mo_tables.reldatabase LEFT JOIN mo_catalog.mo_role ON mo_catalog.mo_tables.owner=role_id ORDER BY reldatabase, relname;
+SELECT mo_catalog.mo_database.datname,mo_catalog.mo_tables.relname,mo_catalog.mo_tables.relkind, if (role_name IS NULL,'-', role_name) AS `owner` FROM mo_catalog.mo_database LEFT JOIN mo_catalog.mo_tables ON mo_catalog.mo_database.datname = mo_catalog.mo_tables.reldatabase LEFT JOIN mo_catalog.mo_role ON mo_catalog.mo_tables.owner=role_id WHERE relname NOT LIKE "__mo_index_secondary_%" ORDER BY reldatabase, relname;
 SELECT relname AS `name`, IF (role_name IS NULL, '-', role_name) AS `owner` FROM mo_catalog.mo_tables LEFT JOIN mo_catalog.mo_role ON mo_catalog.mo_tables.owner=role_id WHERE relkind IN ('v') AND reldatabase='information_schema';
```

Comment:
This is done to remove the secondary index table name containing UUID. I don't think, we can add RegEx here.

Without the WHERE clause, the output would look like this 
```sql
SELECT mo_catalog.mo_database.datname,mo_catalog.mo_tables.relname,mo_catalog.mo_tables.relkind, if (role_name IS NULL,'-', role_name) AS `owner` FROM mo_catalog.mo_database LEFT JOIN mo_catalog.mo_tables ON mo_catalog.mo_database.datname = mo_catalog.mo_tables.reldatabase LEFT JOIN mo_catalog.mo_role ON mo_catalog.mo_tables.owner=role_id ORDER BY reldatabase, relname;
datname    relname    relkind    owner
mo_mo    null    null    -
information_schema    character_sets    r    accountadmin
information_schema    columns    v    accountadmin
information_schema    engines    r    accountadmin
information_schema    key_column_usage    r    accountadmin
information_schema    keywords    r    accountadmin
information_schema    parameters    r    accountadmin
information_schema    partitions    v    accountadmin
information_schema    processlist    v    accountadmin
information_schema    profiling    r    accountadmin
information_schema    routines    r    accountadmin
information_schema    schemata    v    accountadmin
information_schema    statistics    v    accountadmin
information_schema    tables    v    accountadmin
information_schema    triggers    r    accountadmin
information_schema    user_privileges    r    accountadmin
information_schema    views    v    accountadmin
mo_catalog    mo_columns    r    -
mo_catalog    mo_configurations    v    accountadmin
mo_catalog    mo_database    r    -
mo_catalog    mo_increment_columns        accountadmin
mo_catalog    mo_indexes    r    accountadmin
mo_catalog    mo_locks    v    accountadmin
mo_catalog    mo_mysql_compatibility_mode    r    accountadmin
mo_catalog    mo_pubs    r    accountadmin
mo_catalog    mo_role    r    accountadmin
mo_catalog    mo_role_grant    r    accountadmin
mo_catalog    mo_role_privs    r    accountadmin
mo_catalog    mo_sessions    v    accountadmin
mo_catalog    mo_stages    r    accountadmin
mo_catalog    mo_stored_procedure    r    accountadmin
mo_catalog    mo_table_partitions    r    accountadmin
mo_catalog    mo_tables    r    -
mo_catalog    mo_transactions    v    accountadmin
mo_catalog    mo_user    r    accountadmin
mo_catalog    mo_user_defined_function    r    accountadmin
mo_catalog    mo_user_grant    r    accountadmin
mo_catalog    mo_variables    v    accountadmin
mysql    __mo_index_secondary_2f1c14de-6e04-11ee-ba72-723e89f7b972        accountadmin
mysql    __mo_index_secondary_2f1da7d6-6e04-11ee-ba72-723e89f7b972        accountadmin
mysql    __mo_index_secondary_2f20941e-6e04-11ee-ba72-723e89f7b972        accountadmin
mysql    columns_priv    r    accountadmin
mysql    db    r    accountadmin
mysql    procs_priv    r    accountadmin
mysql    tables_priv    r    accountadmin
mysql    user    r    accountadmin
system    statement_info    r    accountadmin
system_metrics    metric    r    accountadmin
system_metrics    server_connections    v    accountadmin
system_metrics    server_storage_usage    v    accountadmin
system_metrics    sql_statement_errors    v    accountadmin
system_metrics    sql_statement_total    v    accountadmin
system_metrics    sql_transaction_errors    v    accountadmin
system_metrics    sql_transaction_total    v    accountadmin
```